admin_manual/installation/apps_supported.rst

```diff
@@ -38,6 +38,7 @@ Below is the list of apps supported for Nextcloud 12. Supported here means that
 * Provisioning API
 * Server info (monitoring app)
 * Sharebymail
+* Socialsharing-plugins (Diaspora, email, Facebook, Google+ and Twitter)
```

Comment:
I understand ... but since this is an open system, there might be other social plugins in the future not supported by GmbH ... but I think this is OK for now 😁